core/src/main/java/com/radixdlt/p2p/addressbook/BerkeleyAddressBookStore.java

```diff
@@ -83,18 +86,27 @@
 public final class BerkeleyAddressBookStore implements AddressBookPersistence {
   private static final Logger log = LogManager.getLogger();
 
+  static {
+    System.setProperty("je.disable.java.adler32", "true");
```

Comment:
I carried this over; it makes BerkeleyDb use its own Adler32 (checksum) impl rather than JDKs, It doesn't seem to be needed though, as per comment in https://github.com/berkeleydb/je/blob/10c9f34f67f487bdfbd7579290f4b2d6438f18c6/src/com/sleepycat/je/utilint/Adler32.java#LL21C5-L21C5 they believe it was fixed in Java 5.

I guess we should just remove it.

Reply:
_(removed)_